### PR TITLE
feat: Introducing Elestio as fully managed deployment option

### LINKED ISSUE #276: Introducing Elestio as one-click deployment option with others
Hey team,
I am Kaiwalya, Developer Advocate at Elestio. Elestio has been providing options of fully deploying and managing Lemmy application [as shown here](https://elest.io/open-source/lemmy). I think it would be a great idea if we can add it to official readme/documentation here.
In addition to this, if you are interested we provide collaboration opportunities with tools we support by revenue share upon addition of this method in docs. If you would like to collaborate, just drop me an email at [kaiwalya@elest.io](mailto:kaiwalya@elest.io) :)

image

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
closes #276

### CODE REVIEW

**Comment by kaiwalyakoparkar** [comment]
@Nutomic Thank you so much for your suggestion. I have made the changes as requested. Also as you get chance, kindly take a look at the proposed collaboration opportunity in #277 🙌

File: src/SUMMARY.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 - [Install with Docker](administration/install_docker.md)
 - [Install with Ansible](administration/install_ansible.md)
 - [Install from Scratch](administration/from_scratch.md)
-- [Install on Elestio](adminitstration/install_elestio.md)
+- [Install on Elestio](adminitstration/)
```
Comment (dessalines) [inline src/SUMMARY.md:21]: Mispelled. Make sure you test this.

File: src/administration/administration.md [hunk 2]
```diff
@@ -17,6 +17,9 @@ We recommend using Ansible, because it simplifies the installation and also make
 
 Lemmy uses roughly 150 MB of RAM in the default Docker installation. CPU usage is negligible.
 
+### Managed Hostings
```
Comment (dessalines) [inline src/administration/administration.md:20]: Hopefully this will fail the prettier CI check.

**Review by dessalines — CHANGES_REQUESTED** [review]

File: src/SUMMARY.md [hunk 3]
```diff
@@ -18,7 +18,7 @@
 - [Install with Docker](administration/install_docker.md)
 - [Install with Ansible](administration/install_ansible.md)
 - [Install from Scratch](administration/from_scratch.md)
-- [Install on Elestio](adminitstration/)
+- [Install on Elestio](administration/)
```
Comment (Nutomic) [inline src/SUMMARY.md:21]: You can get rid of this line as its part of the existing administration page now which is already linked.

**Review by Nutomic — APPROVED** [review]

**Review by dessalines — APPROVED** [review]

**Pull request merged**